src/particle/particle/particle.py

```diff
@@ -402,7 +408,7 @@ def to_list(
             for fld in exclude_fields:
                 try:
                     tbl_names.remove(fld)
-                except:
+                except ValueError:
```

Comment:
No blank excepts! Catches memory errors, Ctrl-C, etc...

Reply:
Hmm, we had been through these before. Damn, it creeped in again!

Reply:
Thats why they need to be checked in CI (as there are now!) :)